hummingbot/strategy/strategy_base.pyx

```diff
@@ -38,3 +43,66 @@ cdef class StrategyBase(TimeIterator):
             past_trades += list(map(lambda ofe: event_to_trade(ofe, market.__class__.__name__), order_filled_events))
 
         return sorted(past_trades, key=lambda x: x.timestamp)
+
+    def market_status_data_frame(self, market_symbol_pairs: List[Tuple[MarketBase, str, str, str]]) -> pd.DataFrame:
```

Comment:
It may be better to make the data type behind the `Tuple[MarketBase, str, str, str]` explicit by putting it to a NamedTuple class, if you expect this would be used across all strategy classes.